cypress/e2e/bookStore.cy.js

```diff
@@ -1,11 +1,67 @@
 /// <reference types='cypress' />
 
 describe('Book Store app', () => {
-  before(() => {
+  const user = {
+    username: 'hywomosamo',
+    password: 'QwertY!8787'
+  };
 
+  const book = {
+    title: 'Speaking JavaScript',
+    author: 'Axel Rauschmayer',
+    publisher: 'O\'Reilly Media',
+    description: 'Like it or not, JavaScript is everywhere these days'
+  };
+  const alertMessage = {
+    added: 'Book added to your collection',
+    deleted: 'Book deleted'
+  };
+
+  beforeEach(() => {
+    cy.visit('/login')
   });
 
-  it('', () => {
+  it('Should provide an ability to login', () => {
+    cy.get('#userName')
+      .type(user.username);
+    cy.get('#password')
+      .type(user.password);
+    cy.get('#login')
+      .click();
+    cy.url()
+      .should('include', '/profile');
+    cy.get('#userName-value')
+      .should('contain', user.username);
+  });
 
+  it.only('Should provite an ability to searh for the "Speaking JavaScript" book, add to collection and delete from collection', () => {
```

Comment:
Should be 2 different tests here: for searching and adding a book and another one for deleting